### DISCUSSION TITLE
How do I handle static assets?

### DISCUSSION THREAD

**Discussion by LaserKaspar** [privatenumber/pkgroll#52] [Q&A] [answered]
### Question

I want to include .json files and they should also be packaged.
I also have some .wasm files wich need to be included.

Some kind of public folder would be great.

My current solution is to copy them over with an external script, is there a better way to do this?

Also I can't use dynamic imports for json-files for some reason, assertion fails. Any ideas how to fix that without changing the target to nodeNext?

**Comment by privatenumber** [comment, marked as answer]
I don't think there's a way to do that now. Feel free to file a feature request!
